apps/point_cloud_editor/src/common.cpp

```diff
@@ -64,7 +64,7 @@ multMatrix(const float* left, const float* right, float* result)
       r[i * MATRIX_SIZE_DIM + j] = sum;
     }
   }
-  std::copy(r, r+MATRIX_SIZE, result);
```

Comment:
Could you check the PR for all such places where you changed `std::copy` to `std::copy_n` and revert those, please? (for the reasons I described in my earlier comment)